functions.php

```diff
@@ -1,8 +1,15 @@
 <?php
+/** 
+* Считает количество задач в проекте 
+* @param array $tasks Массив с задачами
+* @param string $project Название проекта
+* 
+* @return int $task_count Число задач в проекте
+*/
+
 function countTasks($tasks, $project) {
```

Comment:
Между комментарием и функцией не должно быть пустых строк
```suggestion
*/
function countTasks($tasks, $project) {
```